XamarinCommunityToolkit/Behaviors/AttachedNumericValidationBehavior.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors
+{
+    public static class AttachedNumericValidationBehavior
+    {
+        public static readonly BindableProperty AttachBehaviorProperty =
+            BindableProperty.CreateAttached("AttachBehavior", typeof(bool), typeof(AttachedNumericValidationBehavior), false, propertyChanged: OnAttachBehaviorChanged);
+
+        public static bool GetAttachBehavior(BindableObject view)
+        {
+            return (bool)view.GetValue(AttachBehaviorProperty);
+        }
+
+        public static void SetAttachBehavior(BindableObject view, bool value)
+        {
+            view.SetValue(AttachBehaviorProperty, value);
+        }
+
+        static void OnAttachBehaviorChanged(BindableObject view, object oldValue, object newValue)
+        {
+            var entry = view as Entry;
+
+            if (entry == null)
+            {
+                return;
+            }
+
+            bool attachBehavior = (bool)newValue;
```

Comment:
**var** value = (TYPE)obj; // nitpick: no need to write **bool** explicitly twice.